src/Datadog.Trace/Agent/Api.cs

```diff
@@ -117,6 +118,14 @@ public async Task SendTracesAsync(Span[][] traces)
                         return;
                     }
 #endif
+                    var isSocketException = false;
+                    if (ex.InnerException is SocketException se)
+                    {
+                        isSocketException = true;
+                        Log.Error(se, "Unable to communicate with the trace agent.", _tracesEndpoint);
```

Comment:
Does this need an `{Endpoint}` in the message?

Reply:
Actually no, it's not needed, as the previous log statements call that out, I'll remove the property.

Reply:
Actually, I changed my mind, it would be nice to see.